Source/Scene/Expression.js

```diff
@@ -156,6 +182,21 @@ define([
         }
     }
 
+    function checkRelationalTypes(left, right) {
+        var ltype = typeof(left);
+        var rtype = typeof(right);
+        if (ltype !== rtype) {
+            //>>includeStart('debug', pragmas.debug);
```

Comment:
Throughout the `///` statements can go around the entire `if` block (or function implementation in this case).